linera-core/src/chain_worker/state/temporary_changes.rs

```diff
@@ -354,6 +342,30 @@ where
         }
         Ok(ChainInfoResponse::new(info, self.0.config.key_pair()))
     }
+
+    fn check_blob_size(
```

Comment:
If feels like I've already reviewed logic very similar to this (in the recent PR where the blob limit was introduced). Is there no chance to somehow reuse?

Reply:
I extracted this logic from the place where it was before; otherwise it hasn't changed. So I'd say this _is_ the reuse?